README.md

```diff
@@ -5,7 +5,41 @@ Create Ruby clients from a protocol buffer description of an API.
 **Note** This project is a preview. Please try it out and let us know what you think,
 but there are currently no guarantees of stability or support.
 
-## Usage
+## Getting started using the Docker image
+The easiest way to use the generator is to run it with the Docker image. There are several docker images containing various flavors of gapic generators, we are going to use the `gapic-generator-ruby` image which is generating code using the [gapic-generator-cloud](https://github.com/googleapis/gapic-generator-ruby/tree/master/gapic-generator-cloud).
```

Comment:
Why does the `gapic-generator-ruby` image use `gapic-generator-cloud` and not `gapic-generator`?

Reply:
I agree the terminology is a bit confusing. The `gapic-generator-ruby` image being referenced here is specifically `gcr.io/gapic-images/gapic-generator-ruby` which is cloud's image (following the naming convention used by the other cloud microgenerators).